sonarts-core/tests/rules/noInMisuseRule/noInMisuseRule.lint.ts

```diff
@@ -0,0 +1,57 @@
+function nokFunc1() {
+  let arr = ["a", "b", "c"];
+
+  let expectedValue = "b";
+  if (expectedValue in arr) {
+//    ^^^^^^^^^^^^^^^^^^^^    {{Use "indexOf" or "includes" (available from ES2016) instead.}}
+    return expectedValue + " found in the array";
```

Comment:
@andrea-guarino-sonarsource you don't need all this boilerplate from RSPEC in unit tests, keep only meaningful code.